src/graphrag_toolkit/storage/__init__.py

```
@@ -7,4 +7,4 @@
 from .vector_index_factory import VectorIndexFactory
 from .vector_store import VectorStore
 from .vector_store_factory import VectorStoreFactory, VectorStoreType
-from .constants import INDEX_KEY, EMBEDDING_INDEXES
+from .constants import INDEX_KEY, ALL_EMBEDDING_INDEXES, DEFAULT_EMBEDDING_INDEXES
```

Ruff (0.8.2)

`.constants.INDEX_KEY` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)